eng/versioning/version_client.txt

```diff
@@ -67,10 +67,10 @@ com.azure:azure-core-serializer-json-gson;1.1.1;1.2.0-beta.1
 com.azure:azure-core-serializer-json-jackson;1.2.2;1.3.0-beta.1
 com.azure:azure-core-test;1.6.1;1.7.0-beta.1
 com.azure:azure-core-tracing-opentelemetry;1.0.0-beta.9;1.0.0-beta.10
-com.azure:azure-cosmos;4.14.0;4.15.0-beta.1
+com.azure:azure-cosmos;4.15.0-beta.1;4.15.0-beta.2
```

Comment:
Should this dependency version be `4.14.0` and a `beta_` tag be added for locations where the shipped beta will be the dependency version being used.

Reply:
Yeah, we can do that too, but that will just be adding more complexity .. We can treat this as a Stable Beta release, and keep the dependency version as 4.15.0-beta.1
Thoughts ?

Reply:
Dependency versions are supposed to be GA. It's against our version policy to ship an GA package that takes a dependency on a non-GA package.

Reply:
I see, don't want to go in details of GA vs non-GA vs Beta packages :) Have discussed that far too many times :P 
I will update the dependency version to 4.14.0 and will keep the current version to 4.15.0-beta.2
That should be okay, right ?
@alzimmermsft @danieljurek